settings.py

```diff
@@ -752,7 +752,7 @@ def _color_scheme_completions(view):
                 "{}  \tPackage: {}".format(file_name, package),
                 scheme_path
             )
-            completions.add(item)
+            completions.append(item)
         return completions
```

Comment:
Just found some other issues within value_completions(). Will have a deeper look into that.